### PR TITLE
Add 2023 minimum eff. ASHPs to options_lookup.tsv

### PR DESCRIPTION
**Pull request by whiphi92**
## Pull Request Description

Adding 2023 minimum efficiency air-source heat pumps (ASHPs) to the resources/options_lookup.tsv file.

Specifications: 15 SEER 9.0 HSPF
source: https://www.eia.gov/todayinenergy/detail.php?id=40232

Eric Wilson (@ejhw) and I thought it would be a good idea to add this ASHP to the options_lookup.tsv file since it will be a common HVAC unit come 2023. I determined COP by interpolating between HSPF and COP of existing ASHP options.

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing
- [ ] The [changelog](https://github.com/NREL/resstock/blob/develop/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with develop

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: resources/options_lookup.tsv [hunk 1]
```diff
@@ -699,16 +699,19 @@ HVAC Heating Efficiency	"ASHP, SEER 14.3, 8.5 HSPF"	ResidentialHVACAirSourceHeat
...
+		ResidentialAirflow																																													
+		ResidentialHVACSizing																																													
+HVAC Heating Efficiency	"ASHP, SEER 16, 9.0 HSPF"	ResidentialHVACAirSourceHeatPumpSingleSpeed	seer=16.0	hspf=9.0	eer=13.2	cop=3.8	shr=0.73	fan_power_rated=0.365	fan_power_installed=0.5	min_temp=0.0	crankcase_capacity=0.02	crankcase_temp=55.0	eer_capacity_derate_1ton=1.0	eer_capacity_derate_2ton=1.0	eer_capacity_derate_3ton=1.0	eer_capacity_derate_4ton=1.0	eer_capacity_derate_5ton=1.0	cop_capacity_derate_1ton=1.0	cop_capacity_derate_2ton=1.0	cop_capacity_derate_3ton=1.0	cop_capacity_derate_4ton=1.0	cop_capacity_derate_5ton=1.0	heat_pump_capacity=autosize	supplemental_efficiency=1.0	supplemental_capacity=autosize	dse=NA	cop_capacity_derate_1ton=1.0	cop_capacity_derate_2ton=1.0	cop_capacity_derate_3ton=1.0	cop_capacity_derate_4ton=1.0	cop_capacity_derate_5ton=1.0	heat_pump_capacity=autosize	supplemental_efficiency=1.0	supplemental_capacity=autosize	dse=NA
```
Comment (ejhw) [inline resources/options_lookup.tsv:705]: Check the argument lists. Why are they different lengths?

File: resources/options_lookup.tsv [hunk 2]
```diff
@@ -699,16 +699,19 @@ HVAC Heating Efficiency	"ASHP, SEER 14.3, 8.5 HSPF"	ResidentialHVACAirSourceHeat
...
+HVAC Heating Efficiency	"ASHP, SEER 22, 10 HSPF"	ResidentialHVACAirSourceHeatPumpVariableSpeed	seer=22.0	hspf=10.0	eer=17.4	eer2=16.8	eer3=14.3	eer4=13.0	cop=4.82	cop2=4.56	cop3=3.89	cop4=3.92	shr=0.84	shr2=0.79	shr3=0.76	shr4=0.77	capacity_ratio=0.49	capacity_ratio2=0.67	capacity_ratio3=1.0	capacity_ratio4=1.2	fan_speed_ratio_cooling=0.7	fan_speed_ratio_cooling2=0.9	fan_speed_ratio_cooling3=1.0	fan_speed_ratio_cooling4=1.26	fan_speed_ratio_heating=0.74	fan_speed_ratio_heating2=0.92
```
Comment (ejhw) [inline resources/options_lookup.tsv:714]: Why are these lines changing?

File: resources/options_lookup.tsv [hunk 3]
```diff
@@ -834,10 +837,10 @@ HVAC Cooling Efficiency	"AC, SEER 14"	ResidentialHVACCentralAirConditionerSingle
...
-HVAC Cooling Efficiency	"AC, SEER 24.5"	ResidentialHVACCentralAirConditionerVariableSpeed	seer=24.5	eer=19.2	eer2=18.3	eer3=16.5	eer4=14.6	shr=0.98	shr2=0.82	shr3=0.745	shr4=0.77	capacity_ratio=0.36	capacity_ratio2=0.64	capacity_ratio3=1.0	capacity_ratio4=1.16	fan_speed_ratio=0.51	fan_speed_ratio2=0.84	fan_speed_ratio3=1.0	fan_speed_ratio4=1.19	fan_power_rated=0.14	fan_power_installed=0.3	crankcase_capacity=0.0	crankcase_temp=55	eer_capacity_derate_1ton=1.0	eer_capacity_derate_2ton=1.0	eer_capacity_derate_3ton=0.89	eer_capacity_derate_4ton=0.89	eer_capacity_derate_5ton=0.89	capacity=autosize	dse=NA																	
+HVAC Cooling Efficiency	"AC, SEER 24.5"	ResidentialHVACCentralAirConditionerVariableSpeed	seer=24.5	eer=19.2	eer2=18.3	eer3=16.5	eer4=14.6	shr=0.98	shr2=0.82	shr3=0.745	shr4=0.77	capacity_ratio=0.36	capacity_ratio2=0.64	capacity_ratio3=1.0	capacity_ratio4=1.16	fan_speed_ratio=0.51	fan_speed_ratio2=0.84	fan_speed_ratio3=1.0	fan_speed_ratio4=1.19	fan_power_rated=0.14	fan_power_installed=0.3	crankcase_capacity=0.0	crankcase_temp=55	eer_capacity_derate_1ton=1.0	eer_capacity_derate_2ton=1.0	eer_capacity_derate_3ton=0.89																					
 		ResidentialAirflow
```
Comment (ejhw) [inline resources/options_lookup.tsv:844]: Why are these lines changing?

**Review by ejhw — CHANGES_REQUESTED** [review]

File: resources/options_lookup.tsv (hunk 1 above)
Comment (whiphi92) [reply resources/options_lookup.tsv:705]: Application I was using to edit probably did something strange. I'll give it another go.

**Review by ejhw — APPROVED** [review]

**Pull request merged**